security/libprelude/Portfile

```diff
@@ -67,12 +64,6 @@ destroot.keepdirs \
         ${destroot}${prefix}/var/spool/prelude \
         ${destroot}${prefix}/etc/prelude/profile
 
-variant python27 description {python2.7 bindings} {
-        configure.args-replace  --without-python2 \
-                                --with-python2=${prefix}/bin/python2.7
-        depends_lib-append port:python27
-}
```

Comment:
Is it no longer possible to build python27 bindings?

Reply:
Pyython 2 is EOL so I remove the support.